contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -51,32 +52,40 @@ const (
 	Deleted  = FlagType("deleted")
 )
 
+var defaultRoles = []string{"*"}
+
 // A struct that describes a pod task.
 type T struct {
 	ID          string
 	Pod         api.Pod
-	Spec        Spec
+	Spec        *Spec
```

Comment:
Don't like to remove the pointer here. We had the discussion in another context today. Here is makes a difference. There is "no spec" and there is an "empty spec". We shouldn't mix both.